resources/lang/id/rankings.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'header' => 'Peringkat :type',
+    'type' => [
+        'performance' => 'Performa',
+        'charts' => 'Spotlights',
+        'score' => 'Skor',
+        'country' => 'Negara',
+    ],
+    'stat' => [
+        'accuracy' => 'Akurasi',
+        'active_users' => 'Pengguna Aktif',
+        'country' => 'Negara',
+        'play_count' => 'Jumlah Main',
+        'performance' => 'Performa',
+        'total_score' => 'Total Skor',
+        'ranked_score' => 'Skor Ranked',
+        'average_score' => 'Rata-Rata Skor',
+        'average_performance' => 'Rata-Rata Performa',
```

Comment:
Regarding this actually if you looked on the page (http://osu.ppy.sh/rankings/osu/country) you'll see that the space there isn't actually that much (the original English localization even lists "Avg. Perf." instead of "Average Performance"), so perhaps consider simplifying these into stuffs like these : 

+        'performance' => 'Total pp',
+        'average_score' => 'Skor Rata-Rata',
+        'average_performance' => 'pp Rata-Rata',